.readthedocs.yml

```diff
@@ -16,7 +16,7 @@ build:
       - pip install llvmlite~=0.38 --ignore-installed
     pre_build:
       - sphinx-apidoc -o docs/ --private --module-first xclim xclim/testing/tests
-      - env SKIPNOTEBOOKS=1 sphinx-build -b linkcheck docs/ _build/linkcheck
+      - env SKIPNOTEBOOKS=1 sphinx-build -b linkcheck docs/ _build/linkcheck || true
```

Comment:
What does this do? Do I understand it ensures the bash command always returns "successfully" ?

Reply:
Essentially. The example necessitating it here is that dask.org was down today (it recently went back up). We know this is the proper link, so we don't want it to crash. The command here is now more of a verbose warning.